model_templates/python3_unstructured_with_mlops_reporting/datasets/holdout-surgical-dataset.csv

```diff
@@ -0,0 +1,401 @@
+id,bmi,Age,asa_status,baseline_cancer,baseline_charlson,baseline_cvd,baseline_dementia,baseline_diabetes,baseline_digestive,baseline_osteoart,baseline_psych,baseline_pulmonary,ahrq_ccs,ccsComplicationRate,ccsMort30Rate,complication_rsi,dow,gender,hour,month,moonphase,mort30,mortality_rsi,race,complication
```

Comment:
We typically put datasets into some tests/testdata directory but I dont like it. I dont like that pattern too much because dataset lives too far, but take to account that if it lives right near the model, people will try to upload the whole directory along with this dataset as model filles.

Reply:
Yeah, this is why I put them in a dedicated directory. 
In the README.md I specify the model's files and also that the datasets are for testing. 
I can add more information there ...